diff -urN libgnomekbd-2.19.91/capplet/gkbd-indicator-plugins-capplet.desktop.in.in libgnomekbd-2.19.91.new/capplet/gkbd-indicator-plugins-capplet.desktop.in.in
--- libgnomekbd-2.19.91/capplet/gkbd-indicator-plugins-capplet.desktop.in.in	2007-08-29 17:58:10.000000000 +0200
+++ libgnomekbd-2.19.91.new/capplet/gkbd-indicator-plugins-capplet.desktop.in.in	2007-08-29 18:01:35.000000000 +0200
@@ -7,7 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;Settings;DesktopSettings;
+Categories=GTK;GNOME;Settings;DesktopSettings;
 OnlyShowIn=GNOME;
 NoDisplay=true
 X-GNOME-Bugzilla-Bugzilla=GNOME
